### PR TITLE
JAVA-37857-Fix-formatting-of-POMs

### PR DESCRIPTION
**Pull request by vunamtien**

### CODE REVIEW

File: spring-cloud-modules/pom.xml [hunk 1]
```diff
@@ -18,7 +18,7 @@
 
     <modules>
         <module>api-gateway</module>
-	<module>spring-cloud-loadbalancer</module>
```
Comment (anastasiosioannidis) [inline spring-cloud-modules/pom.xml:21]: is there an indentation below?
If yes, we don't need it.

Comment (vunamtien) [reply spring-cloud-modules/pom.xml:21]: i fixed it, thanks!

File: spring-cloud-modules/spring-cloud-sleuth/pom.xml
Comment (anastasiosioannidis) [inline spring-cloud-modules/spring-cloud-sleuth/pom.xml:1]: artifactId should be below groupId, correct?

File: spring-security-modules/spring-security-authorization/spring-security-annotation-template-parameter/pom.xml [hunk 2]
```diff
@@ -1,15 +1,19 @@
 <?xml version="1.0" encoding="UTF-8"?>
```
Comment (anastasiosioannidis) [inline spring-security-modules/spring-security-authorization/spring-security-annotation-template-parameter/pom.xml:1]: can you remove identation of lines 3,4 below?

File: spring-swagger-codegen/openapi-custom-generator/pom.xml [hunk 3]
```diff
@@ -1,17 +1,17 @@
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```
Comment (anastasiosioannidis) [inline spring-swagger-codegen/openapi-custom-generator/pom.xml:1]: please remove identation of line 2 below

File: spring-cloud-modules/spring-cloud-sleuth/pom.xml
Comment (vunamtien) [reply spring-cloud-modules/spring-cloud-sleuth/pom.xml:1]: i fixed it, thanks!

File: spring-security-modules/spring-security-authorization/spring-security-annotation-template-parameter/pom.xml (hunk 2 above)
Comment (vunamtien) [reply spring-security-modules/spring-security-authorization/spring-security-annotation-template-parameter/pom.xml:1]: i fixed it, thanks!

File: spring-swagger-codegen/openapi-custom-generator/pom.xml (hunk 3 above)
Comment (vunamtien) [reply spring-swagger-codegen/openapi-custom-generator/pom.xml:1]: i fixed it, thanks!

**Pull request merged**